lib/irb/debug.rb

```diff
@@ -57,6 +57,22 @@ def DEBUGGER__.capture_frames(*args)
           DEBUGGER__::ThreadClient.prepend(SkipPathHelperForIRB)
         end
 
+        unless DEBUGGER__::CONFIG[:no_hint]
+          irb_output_modifier_proc = Reline.output_modifier_proc
+
+          Reline.output_modifier_proc = proc do |output, complete:|
+            unless output.strip.empty?
+              cmd = output.split(/\s/, 2).first
+
+              if DEBUGGER__.commands.key?(cmd)
+                output = output.sub(/\n$/, " # debug command")
```

Comment:
`output = output.sub(/\n$/, " # debug command\n")`

The line count is changed in this operation.
IRB crashes when you type `info[OPTION+ENTER]`
```
irb(main):001> debug
irb:rdbg(main):002> info【OPTION+ENTER】
/Users/tomoya.ishida/.rbenv/versions/3.2.2/lib/ruby/gems/3.2.0/gems/reline-0.4.0/lib/reline/line_editor.rb:1128:in `+': no implicit conversion of nil into String (TypeError)
```

Reply:
Good catch, thank you!